models.py

```diff
@@ -165,6 +165,19 @@
 MINOR_EXISTING_CHANGES = 4
 EXTREMELY_SMALL_CHANGE = 5
 
+# Status for security and privacy reviews.
+REVIEW_PENDING = 1
+REVIEW_IN_PROGRESS = 2
+REVIEW_COMPLETED = 3
+REVIEW_NA = 4
+
+REVIEW_STATUS_CHOICES = {
+    REVIEW_PENDING: 'Pending',
+    REVIEW_IN_PROGRESS: 'In progress',
+    REVIEW_COMPLETED: 'Completed',
+    REVIEW_NA: 'Not applicable',
```

Comment:
Can tag review default to this for if a spec implementation is being completed or a feature is being deprecated/removed?

Reply:
Done.